.github/ISSUE_TEMPLATE/enchancement.yml

```diff
@@ -1,5 +1,5 @@
-name: Feature Request
-description: Request a new feature or improvement specific to this CORTX component. Please check first to make sure this feature has not yet been requested.  
+name: Feature request
```

Comment:
Suggestion: "Feature request or Improvement suggestion"

Reply:
@keithpine , you OK with this?

Reply:
I don't have a preference either way.